pyproject.toml

```diff
@@ -0,0 +1,100 @@
+[project]
+name = "exoctk"
+description = "Exoplanet Characterization Toolkit"
+readme = "README.md"
+authors = [
+  { name= "Matthew Bourque" },
+  { name="Néstor Espinoza" },
+  { name="Joseph Filippazzo"},
+  { name="Mees Fix" },
+  { name="Teagan King" },
+  { name="Catherine Martlin" },
+  { name="Jennifer Medina" },
+  { name="Natasha Batalha" },
+  { name="Michael Fox" },
+  { name="Jules Fowler" },
+  { name="Jonathan Fraine" },
+  { name="Matthew Hill" },
+  { name="Nikole Lewis" },
+  { name="Kevin Stevenson" },
+  { name="Jeff Valenti" },
+  { name="Hannah Wakeford" },
+]
+keywords = ["astronomy", "python"]
+classifiers = ["Programming Language :: Python"]
+dependencies = [
+  'asteval',
+  'astropy',
+  'astroquery',
+  'bandit',
+  'batman-package',
+  'bibtexparser',
+  'bokeh',
+  'boto3',
+  'corner',
+  'cython',
+  'docopt',
+  'docutils',
+  'flake8',
```

Comment:
I don't think this is needed in the wheel dependencies (unless it is directly imported by the application at runtime). Are there other dependencies like this that can be removed?
```suggestion
```